codebuild/fuzz_codebuild.config

```diff
@@ -116,10 +116,10 @@ start_time: 05:00
 build_job_name: s2nFuzzScheduled
 input: {"environmentVariablesOverride": [{"name": "FUZZ_TESTS","value": "s2n_hybrid_ecdhe_sike_r1_fuzz_test"}]}
 
-[CloudWatchEvent:s2n_hybrid_ecdhe_sike_r2_fuzz_test]
+[CloudWatchEvent:s2n_hybrid_ecdhe_sike_r3_fuzz_test]
```

Comment:
We should confirm with @dougch that editing this Codebuild config is all that's needed. There may be some other changes needed also.